tests/console/rmt/rmt_feature.pm

```diff
@@ -90,7 +100,17 @@ sub run {
         die 'cleanup repos failed';
     }
 
+    record_info('add custom repo');
+    assert_script_run("rmt-cli repos custom add https://download.opensuse.org/repositories/games:/tools/SLE_15_SP3/x86_64/ Games");
+    assert_script_run("rmt-cli repos custom list | grep Games");
+    # attach the custom repo to a product
+    assert_script_run("rmt-cli products enable sle-module-development-tools/15.3/x86_64");
+    assert_script_run("rmt-cli repos custom attach games 2161");
```

Comment:
What's the meaning of '2161'?

Reply:
2161 is the product number of sle-module-development-tools/15.3/x86_64.

Reply:
If so please add a comments here for the '2161', thanks.

Reply:
Based on Huajian's suggestion, I updated the code here, please help to review, thanks.